crypto/compat/arc4random_irix.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2025 Kazuo Kuroi (kazuo@irixnet.org)
```

Comment:
Most of this file is a copy of the other `arc4random_*.h`, so the original authors should be retained:

```suggestion
 * Copyright (c) 1996, David Mazieres <dm@uun.org>
 * Copyright (c) 2008, Damien Miller <djm@openbsd.org>
 * Copyright (c) 2013, Markus Friedl <markus@openbsd.org>
 * Copyright (c) 2014, Theo de Raadt <deraadt@openbsd.org>
 * Copyright (c) 2025 Kazuo Kuroi (kazuo@irixnet.org)
```